test/integration/user_can_checkout_test.rb

```diff
@@ -8,19 +8,19 @@ class UserCanCheckoutTest < ActionDispatch::IntegrationTest
     visit "/items"
     click_on "Add to Cart"
 
-    within ".navbar" do
-      click_on "Cart"
-    end
-
+    # within ".navbar" do
+    #   click_on "Cart"
+    # end
+    find(:xpath, "//a[@href='/cart']").click
```

Comment:
Its a way to find a link by the path itself vs. just clicking on a link. I had to use this because our cart is an image + dynamic number